packages/design-system/src/styles/components/_TextField.scss

```diff
@@ -7,6 +7,7 @@
   border-radius: var(--text-input__border-radius);
   color: var(--text-input__color);
   display: block;
+  line-height: var(--text-input__line-height); // Needed to overwrite user agent heading tag styles
```

Comment:
How did you discover this one?

Reply:
This was an artifact me me trying to resolve all the descending specificity lint rules. It was big mad we were defining ds-c-field styles (it was combined with the ds-c-field-mask rule) after `:-moz-ui-invalid` and `:disabled` were defined, so I split the two up and put line height with the rest of the ds-c-field styles. I could revert it, but it seemed nice having all styles associated with a given class in 1 section.

Reply:
Oh I see now 👍